tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreSymbols.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>DynamoVisualProgramming.DynamoCoreSymbols</id>
+    <version>@VERSION@</version>
+    <authors>Autodesk</authors>
+    <owners>Autodesk</owners>
+    <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
```

Comment:
also - I think this license is wrong... should be Apache 2.0

Reply:
hmm How should we confirm? Is this licence for DynamoCore?

Reply:
@mjkkirschner Does it mean we need to update the licence for all of our nuget packages? https://github.com/DynamoDS/Dynamo/blob/master/tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec#L8

Reply:
I think we should for anything we publish to public nuget. But that does not need to happen in this PR.

Reply:
Updated all of them